packages/lesswrong/components/ea-forum/users/DisplayNameWithMarkers.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { registerComponent, Components } from "../../../lib/vulcan-lib";
+
+const styles = (_theme: ThemeType) => ({
+  tooltipPopper: {
+    maxWidth: 200,
+    textAlign: 'center'
+  },
+});
+
+/**
+ * Wrap certain special characters with a tooltip explaining them
+ */
+const DisplayNameWithMarkers = ({ name, classes }: { name: string; classes: ClassesType<typeof styles> }) => {
+  const { LWTooltip } = Components;
+
+  // Show a tooltip if they have the 🔸 or 🔹 at the end of their profile
+  const tenPercentPledgeIndex = name.lastIndexOf("🔸");
+  const trialPledgeIndex = name.lastIndexOf("🔹");
```

Comment:
It'd be nice if this was more declarative so we didn't have to repeat code, and to make it easier to add/remove more options in the future.

Something like:
```
const markers = [
  {text: "🔸", tooltip: (name) => `${name} has taken the 🔸10% Pledge`},
  {text: "🔹", tooltip: (name) => `${name} has taken the 🔹Trial Pledge`},
];
```
that we can then loop through.